doc/ansible/dev/advanced_options.md

```diff
@@ -50,3 +50,31 @@ You may want to manage what your operator watches and the owner references. This
 ```
 ENTRYPOINT ["/usr/local/bin/entrypoint", "--inject-owner-ref=false"]
 ```
+
+## Max Workers
+
+Increasing the number of workers allows events to be processed
+concurrently, which can improve reconciliation performance.
+
+Worker maximums can be set in two ways. Operator **authors** can set the
+default by including extra args to the operator container in
+`operator.yaml`. (Otherwise, the default is 1 worker.)
+
+``` yaml
+- name: operator
+  image: "quay.io/asmacdo/memcached-operator:v0.0.0"
+  imagePullPolicy: "Always"
+  args:
+    - "--max-workers"
+    - "3"
+```
+
+Operator **admins** can override the value by setting an environment
+variable in the format `WORKER_<kind>_<group>`. This variable must be
+all uppercase, and periods are replaced with underscores.
+
+Example:
+
+``` bash
+export WORKER_MEMCACHED_CACHE_EXAMPLE_COM=4
```

Comment:
The way that admins will set the workers is by adding the environment variable to the `env` section of the `operator` container defined in `operator.yaml`, as admins will have control over the deployed manifests. OLM will also allow admins to override the `env` section of a container.